lib/CXGN/BrAPI/v2/Locations.pm

```diff
@@ -30,8 +31,8 @@ sub search {
 	my $location_names_arrayref  = $params->{locationNames } || ($params->{locationNames } || ());
 	my $location_types_arrayref = $params->{locationType} || ($params->{locationTypes} || ());
 
-    if (($institute_names_arrayref && scalar($institute_names_arrayref)>0) || ($coordinates && scalar($coordinates)>0 )|| ( $externalreference_sources_arrayref && scalar($externalreference_sources_arrayref)>0) || ($externalreference_ids_arrayref && scalar($externalreference_ids_arrayref)>0)){
-        push @$status, { 'error' => 'The following search parameters are not implemented: instituteNames, coordinates, externalReferenceID, externalReferenceSource' };
+    if (($institute_names_arrayref && scalar($institute_names_arrayref)>0) || ($coordinates && scalar($coordinates)>0 )|| ( $externalreference_sources_arrayref && scalar($externalreference_sources_arrayref)>0)){
+        push @$status, { 'error' => 'The following search parameters are not implemented: instituteNames, coordinates'};
```

Comment:
Is this check supposed to be here still?

`( $externalreference_sources_arrayref && scalar($externalreference_sources_arrayref)>0)`

If so, should it be represented in the message?

Reply:
Good catch, missed that one

Reply:
Pushed changes